content/event-platform/logs.md

```diff
@@ -0,0 +1,99 @@
+## Logs API Overview
+
+The **Logs** functionality allows you to retrieve a detailed and paginated list of logs from the Event Platform, providing transparency and traceability for events processed within the system. Using the Logs API, you can filter logs based on multiple parameters, such as time range, type of log, or specific identifiers, to easily diagnose issues or audit activity.
+
+### Key Features
+- **Pagination Support**: Retrieve logs page by page for efficient handling of large datasets.
+- **Customizable Filters**: Narrow down results using filters like date range, log type (e.g., "action" or "error"), correlation ID, or subscriber and subscription IDs.
+- **Enhanced Traceability**: Each log entry includes metadata to track events and troubleshoot effectively.
+
+### How to Use the Logs API
+To see the full list of available query parameters and example requests, refer to the **[Logs API Documentation](https://storage.googleapis.com/akecld-prd-sdk-aep-prd-api-assets/openapi_specification.html#tag/Logs/operation/get_logs)**.
+
+### Log Types Explained
+
+The `log_type` field specifies the nature of the log. The two main types are:
+
+#### 1. **Action Logs**
+Action logs represent events performed by either the system or the user. These logs indicate successful operations, such as updates, deletions, or other actions.
+
+**Example**:  
+A user suspends a subscription.
+
+```json
+{
+  "log_id":"0193642e-1083-7e37-93c1-04df60e0f5b6",
+  "log_type":"action",
+  "timestamp":"2024-11-25T16:36:00Z",
+  "operation":"subscription_suspended_by_the_user",
+  "subscriber_id":"019363c4-6e22-7dd4-a811-6aab07f1d474",
+  "subscription_id":"019363c6-139b-7412-a020-75934c917645",
+  "correlation_id":"no_correlation_id",
```

Comment:
do we expose it ? I mean, not only removing it from the documentation but in the real response too !